cms/templates/js/visibility-and-access-editor.underscore

```diff
@@ -0,0 +1,78 @@
+<form>
+<%
+var groupAccess = xblockInfo.get('group_access');
+var selectablePartitions = [];
+var groupIdToName = {};
+contentGroups.forEach(function (partition) {
+    if (partition['scheme'] != 'cohort' && partition['scheme'] != 'enrollment_track') {
+        return;
+    }
+    if (partition['scheme'] == 'enrollment_track') {
+        var tracks = [];
+        var trackSelected = false;
+        partition['groups'].forEach(function (group) {
+            if (group['selected'] === true) {
+                trackSelected = true;
+            }
+        })
+        if (partition['groups'].length <= 1 && trackSelected === false) {
+            return;
+        }
+    }
+    selectablePartitions.push(partition);
+})
+selectablePartitions.forEach(function (partition) {
+    var schemeId = partition['id'];
+    var groups = partition['groups'];
+    groups.forEach(function (group) {
+        var groupId = group['id'];
+        var groupName = group['name'];
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        groupIdToName[fullId] = groupName;
+    })
+})
+%>
+<% if (xblockInfo.isVertical()) { %>
+    <h3 class="modal-section-title">
+        <%- gettext('Unit Access') %>
+    </h3>
+    <div class="modal-section-content access-change">
+        <ul class="list-fields list-input">
+            <li>
+            <%- gettext('Restrict access to:') %>
+            <select class="user-partition-select">
```

Comment:
The select has no label associated with it. This is an a11y violation, as you can see in your browser if you run the accessibility audit tool in Chrome dev tools.

Please look at visibility_editor.html for how you put a label around the select.
Also remove the ul/li structure in here, as it doesn't make sense.

![image](https://user-images.githubusercontent.com/484484/27235215-e9feef9a-528d-11e7-9ecd-c34de81bb893.png)